### PR TITLE
[JOSS] doc: minor language edits of the paper

### PR DESCRIPTION
**Pull request by sappelhoff**
Hi @andportnoy @turbach I am going to review your submission of `fitgrid` to JOSS. I'll link each issue and PR that I open back to the review thread https://github.com/openjournals/joss-reviews/issues/3293 so that we can have an overview of my requests / questions / hints / ... and check back later in how far they have been addressed or should be dropped.

In this first PR I make some stylistic suggestions (mostly some fixes) after reading the paper.

I'll also try to make some comments in the diff where I think you should clarify the paper. GitHub sometimes only allows inline comments close to lines that have been "touched", so I may simply make another post below this one, if I cannot accommodate a comment in inline text.

### CODE REVIEW

File: joss/paper.md [hunk 1]
```diff
@@ -86,17 +86,19 @@ for a large scale experiment. The combinatorial explosion is
...
-been gaining traction for EEG and MEG analysis, as in MNE Python
-@GramfortEtAl2013 and for regression ERPs in R,
+been gaining traction for EEG and MEG analysis, as in MNE-Python
+@GramfortEtAl2013, and for regression ERPs in R
 @TreNew2015. Nevertheless, widely accessible implementations for rERP
-modeling in the Python remain limited.  Development of N. J. Smith's
+modeling in Python remain limited. Development of N. J. Smith's
 promising rERPy Python package for ERP and rERP analysis appears to
-have halted in Python 2.X. MNE Python implements a `linear_regression`
+have halted in Python 2, which reached its end of life in January 2020.
```
Comment (sappelhoff) [inline joss/paper.md:100]: would be nice to get a link to that project here (even if apparently outdated)

File: joss/paper.md [hunk 2]
```diff
@@ -118,7 +120,7 @@ parallel if supported by hardware). The fit results across times and
 channels are available on demand with the same syntax used to access
 results in a single fit object and the results are returned as tidy
 indexed `pandas.DataFrames` for further analysis, visualization, and
```
Comment (sappelhoff) [inline joss/paper.md:122]: would be nice to cite pandas (the paper), and also numpy and other **crucial** packages that you depend on.

File: joss/paper.md [hunk 3]
```diff
@@ -127,12 +129,12 @@ Python vignettes that illustrate their use. While the origins of
...
-which event-related regression modeling is appropriate.  `fitgrid`
+which event-related regression modeling is appropriate. `fitgrid`
 enables researchers to conduct this type of computationally-intensive
 modeling flexibly, efficiently, informatively, and reproducibly with
 familiar scientific computing tools and minimal programming. These
 features make `fitgrid` well-suited for general use in exploratory
-data analysis (EDA), e.g., @UrbDelChaKut2020 and @TroUrbKut2020.
+data analysis (EDA; e.g., @UrbDelChaKut2020 and @TroUrbKut2020).
 
 ![fitgrid TL; DR](fitgrid_overview.png)
```
Comment (sappelhoff) [inline joss/paper.md:139]: I would prefer you add a proper figure caption here. I am not sure in how far the format permits that ... but the "TL; DR" is a bit too brief.

File: joss/paper.md [hunk 4]
```diff
@@ -166,14 +168,14 @@ The `fitgrid` documentation is available online:
 
   ![Downloadable Examples Gallery](examples_gallery.png)
```
Comment (sappelhoff) [inline joss/paper.md:169]: same as above: Maybe slightly expand the figure caption.

File: joss/paper.md [hunk 5]
```diff
@@ -166,14 +168,14 @@ The `fitgrid` documentation is available online:
...
-system recommendations.  The latest stable release of `fitgrid` and
+system recommendations. The latest stable release of `fitgrid` and
 the bleeding edge pre-release development version are packaged for
 Python 3.6, 3.7, and 3.8 on x86_64 linux and distributed on Anaconda
 Cloud. Installation of the stable release into a fresh conda virtual
-environment along with other compatible packages, e.g., `jupyter` for
+environment along with other compatible packages, such as `jupyter` for
 running Example Gallery notebooks, is recommended like so:
 
 ```bash
```
Comment (sappelhoff) [inline joss/paper.md:181]: why are these installation instructions different from the README? especially with regards to the conda channels.

**Review by sappelhoff — COMMENTED** [review]
- [ ] would be nice to add an ORCID ID for `Andrey S. Portnoy`, unless they don't have one.
- [ ] there is a part: `and package pytests on` in the paper: What are package pytests? Tests of the package using `pytest`? I suggest rewording this
- [ ] The summary ends with ` With the shift from averaging to regression modeling, however,
comes a new problem.` I suggest to add one more super brief sentence after this that outlines the problem that is then elaborated in the next section.
- [ ] The part of the paper that ends with `The combinatorial explosion is unavoidable; `fitgrid` contains it.` could be improved by a short extension of that sentence: How (in essence) does fitgrid contain it ... before you go on to elaborate in the next paragraph

**Comment by sappelhoff** [comment]
- [ ] Please also go over all citations and make sure each one has a DOI (whenever possible). You have several without a DOI, and a brief check showed me that at least one of them **does** have a DOI: https://www.hindawi.com/journals/cin/2011/879716/

File: joss/paper.md (hunk 5 above)
Comment (turbach) [reply joss/paper.md:181]: Yes this is a serious problem, the error is mine. To make matters worse, recent changes in upstream packages have impacted the dependencies and broke our CI. I am working on fixes for that and cleaning up the installation instructions now. Will do a patch release to 0.5.1 asap so the README, online docs, and JOSS ms installation instructions are consistent and correct. In the meantime, if you like I can post instructions for installing the pre-release version that at last check installs and runs the examples on x86_64 linux and Intel Macs.

Comment (sappelhoff) [reply joss/paper.md:181]: I see, thanks. I have opened https://github.com/kutaslab/fitgrid/issues/196 to ask some more questions about the installation. As for testing -> I'll use the stable version, which I successfully installed yesterday (but haven't looked into it more than that until now)

Comment (turbach) [reply joss/paper.md:181]: Ok, for the default install you may run into issues with lmer or the parallel processing depending on what versions of what you ended up with. The pre-release is testing OK, let me know if you want to take it for a spin. Thx.

File: joss/paper.md (hunk 1 above)
Comment (turbach) [reply joss/paper.md:100]: Done

File: joss/paper.md (hunk 2 above)
Comment (turbach) [reply joss/paper.md:122]: Done, the References now include pandas, numpy, matplotlib, statsmodels, pymer4, rpy2, R, lme4, lmerTest.

File: joss/paper.md (hunk 3 above)
Comment (turbach) [reply joss/paper.md:139]: Done

File: joss/paper.md (hunk 4 above)
Comment (turbach) [reply joss/paper.md:169]: Done

**Review by sappelhoff — COMMENTED** [review]
thanks for addressing all these points @turbach --> I just see the comments coming in. It would be tremendously helpful for me if - instead of just telling me that it's done - you could supply the link to a commit **where** this was addressed.

Example: eadf358
